### ISSUE TITLE
Unable to resolve module `AccessibilityInfo`

### ISSUE THREAD

**Issue by praba1989** [leksyib/react-native-video-component#1]
Error: Unable to resolve module `AccessibilityInfo` from `node_modules/react-native-video-component/node_modules/react-native/Libraries/react-native/react-native-implementation.js`: AccessibilityInfo could not be found within the project.
 
  "react-native": "^0.63.2",
